src/app/file/ase_format.cpp

```diff
@@ -1016,8 +1017,14 @@ static void ase_file_write_cel_chunk(FILE* f,
       link = nullptr;
   }
 
+  bool saveRawCels = ui::await<bool>([]() -> bool {
+    auto& pref = Preferences::instance();
+    return pref.asepriteFormat.saveRawCels();
+  });
+
```

Comment:
Please don't access `Preferences` from file encoders/decoders (not even using threads). There is an already established mechanism for this:
1. If it's an option to be asked from the UI each time we save we can access preferences from `onAskUserForFormatOptions()` and that goes to the `FileOptions` instance.
2. Use `FileOpConfig` and `FileOpConfig::fillFromPreferences()`

Probably this requires a combination of both where the `AsepriteOptions` is modified (or created) when saving the file depending on the preferences (not asking the user when saving, or in `onAskUserForFormatOptions()` without showing UI, just accessing the `Preferences` instance and updating the `AsepriteOptions`).